### PR TITLE
Schedule view rough

### PR DESCRIPTION
**Pull request by jayp822**
https://github.com/user-attachments/assets/72570cd5-f51d-4e04-b68d-d57f5ef9032d

### CODE REVIEW

File: frontend/package.json [hunk 1]
```diff
@@ -11,10 +11,13 @@
     "lint:check": "eslint --ext \".ts,.vue\" --ignore-path .gitignore .",
     "lint:fix": "eslint --ext \".ts,.vue\" --ignore-path .gitignore --fix .",
     "format:check": "prettier --check .",
-    "format:fix": "prettier --write --list-different ."
+    "format:fix": "prettier --write --list-different .",
+    "fl": "prettier --write --list-different . && eslint --ext \".ts,.vue\" --ignore-path .gitignore --fix ."
```
Comment (mykalbayouk) [inline frontend/package.json:15]: What is the purpose of this?

File: frontend/src/app/components/ui/EmblaCarouselDotButton.tsx
Comment (mykalbayouk) [inline frontend/src/app/components/ui/EmblaCarouselDotButton.tsx:1]: Could we get some comments on this file plz

File: frontend/src/styles/globals.css
Comment (mykalbayouk) [inline frontend/src/styles/globals.css:1]: what is the specific reason to have these css attributes in the global.css file, are they reused outside of a single component?

File: frontend/package.json (hunk 1 above)
Comment (jayp822) [reply frontend/package.json:15]: I used it to format and lint at the same time while working. It can be deleted.

**Pull request closed without merging**